### ISSUE TITLE
Feature Request: Zen Twilight option 

### ISSUE THREAD

**Issue by BlueFox1616** [0xc000022070/zen-browser-flake#16]
The title

**Comment by luisnquin** [comment]
It's done @ https://github.com/0xc000022070/zen-browser-flake/commit/8c53d0c31957f8f91c604bf88f3c322ea6889d63, you have that version available in `packages."${system}".twilight`.

I need to go! So I'll update the README later or tomorrow... I don't know!

**Comment by luisnquin** [comment]
@BlueFox1616 

It's done @ https://github.com/0xc000022070/zen-browser-flake/commit/8c53d0c31957f8f91c604bf88f3c322ea6889d63, you have that version available in `packages."${system}".twilight`.

I need to go! So I'll update the README later or tomorrow... I don't know!

**Comment by BlueFox1616** [comment]
@luisnquin there is an error with it 
`
       error: hash mismatch in file downloaded from 'https://github.com/zen-browser/desktop/releases/download/twilight/zen.linux-x86_64.tar.bz2':
         specified: sha256:0j5dy58kammrz56j3id149k3kdnc0b2y7h03yq6l1n2fpklxq2kc
         got:       sha256:0yhlkyj87fylmjr8zfqv73ljx99wry8j07r19bl88078pll4ifn2
`

**Comment by BlueFox1616** [comment]
i made a pull request and changed it to the correct sha256 unless there is something else that's wrong 
#17

**Comment by luisnquin** [comment]
Yeah, a few hours ago they created a new release under the same tag, I need to start using the commit SHA1 instead.

**Comment by BlueFox1616** [comment]
i installed twillight but it doesnt appear anywhere i also have the beta which is the only app i can find

**Comment by luisnquin** [comment]
Try this:

```bash
$ ls /nix/store | grep twilight
```

And find the binary, maybe I should add an alias for twilight too.

**Comment by luisnquin** [comment]
@BlueFox1616 

I just created a symlink that acts as an alias for the binary so if you choose the twilight version you have two commands available in your $PATH, `zen` and `zen-twilight` unless you mean that it should be exposed with a different name in the .desktop file.

**Comment by BlueFox1616** [comment]
It should be exposed as a different app on the desktop. It even has a diff icon originally.

**Comment by luisnquin** [comment]
@BlueFox1616 

I think this is fixed @ https://github.com/0xc000022070/zen-browser-flake/commit/92c1e51d80641441c1457ef4be640841a857e9bb, please check if that works as expected.

**Comment by BlueFox1616** [comment]
it doesnt work . Unless im doing something wrong 
![Screenshot From 2024-12-30 20-22-34](https://github.com/user-attachments/assets/45f81712-6d1a-4700-8a51-85ee85566c87)

**Comment by luisnquin** [comment]
Given your command line, you should execute `zen-twilight` instead of `zen twilight`.

By the other side, that's not my use case because I'm only using the Twilight version but in your system you have two .desktop files that are pointing to the `zen` executable that I think by default in your case is the **beta version**. I will make those files to directly point to zen-beta and zen-twilight.

**Comment by luisnquin** [comment]
And your icons between both versions are not changing in your system because both packages are providing them under the same name: zen.

**Comment by luisnquin** [comment]
I fixed both issues with the .desktop files. Let me know if that worked for you. ;)

https://github.com/0xc000022070/zen-browser-flake/commit/04058273f2f8c359a9495cf176f707d8c3e5c85f

**Comment by BlueFox1616** [comment]
still treats zen twilight as if they were the same up not just the icon when i open zen twilight it isn't distinct from a zen browser window and on the about tab it seems to still just be the beta

**Comment by vaibd** [comment]
Same error
` error: hash mismatch in file downloaded from 'https://github.com/zen-browser/desktop/releases/download/twilight/zen.linux-x86_64.tar.bz2':
         specified: sha256:0v6747qd0s2znm17wzjl3g69m7v7slx5n8gzr1pcr0zpnjgdq37s
         got:       sha256:0d384ygs47ck8d2cpvismy8dmcbw4aqavqfgpsfhjz8ir8197g6n`

**Comment by BlueFox1616** [comment]
Zen twilight officially works fine . Thanks @luisnquin
